ViteMaDose/ViewModels/Home/HomeViewModel.swift

```diff
@@ -119,13 +119,15 @@ class HomeViewModel {
         let allCentresViewModel = HomeCellStatsViewData(.allCentres(allCountiesStats.total))
         let centresWithAvailabilitiesViewModel = HomeCellStatsViewData(.centresWithAvailabilities(allCountiesStats.disponibles))
         let allAvailabilitiesViewModel = HomeCellStatsViewData(.allAvailabilities(allCountiesStats.creneaux))
+        let percentageAvailabilitiesViewModel = HomeCellStatsViewData(.percentageAvailabilities(allCountiesStats.pourcentage))
         let externalMapViewModel = HomeCellStatsViewData(.externalMap)
 
         statsCell = [
             .title(statsTitleViewModel),
             .stats(allCentresViewModel),
             .stats(centresWithAvailabilitiesViewModel),
             .stats(allAvailabilitiesViewModel),
+            .stats(percentageAvailabilitiesViewModel),
```

Comment:
Possible de changer l'ordre pour ça ? ⬇️ 
```suggestion
            .stats(allAvailabilitiesViewModel),
            .stats(centresWithAvailabilitiesViewModel),
            .stats(percentageAvailabilitiesViewModel),
```

![image](https://user-images.githubusercontent.com/6460866/115444718-da420880-a20c-11eb-8931-d7fdc65c8fb8.png)